lib/src/extensions/widget_extensions.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/widgets.dart';
+
+extension WidgetPaddingX on Widget {
```

Comment:
Wow, it's cool!
I was thinking about doing this this weekend, I'm glad you thought about it.

Reply:
Thanks!